packages/combobox/src/Combobox.story.tsx

```diff
@@ -113,15 +113,35 @@ export default StoryMeta({
   },
 });
 
-const ComboboxOptions = [
+const getComboboxOptions = (withGlyphs = true) => [
```

Comment:
did you test how this renders on .design?

Reply:
thanks for reminding me, i need to do this!

Reply:
I'm not able to render the descriptions and new glyphs on .design but its not blocking. I can dig deeper into this when i work on .design next week.

Reply:
have a feelign that it's related to children being rendered from a function, but i could be wrong